include/lsst/sphgeom/Box.h

```diff
@@ -53,7 +53,7 @@ namespace sphgeom {
 /// - Box::allLongitudes().contains(b.getLon())
 class Box : public Region {
 public:
-    static constexpr uint8_t TYPE_CODE = 'b';
+    static uint8_t const TYPE_CODE;
```

Comment:
This is sad, if it really can't work this way. `constexpr` really should be the way to go.